gpustat/cli.py

```diff
@@ -69,7 +114,12 @@ def main(*argv):
         pass
     # arguments to gpustat
     import argparse
+    try:
+        import shtab
+    except ImportError:
+        from . import _shtab as shtab
```

Comment:
Why do we want to provide `_shtab` fallback when `shtab` is not installed? (please see the errors around L189)  Would you want to make `shtab` as an optional dependency?

Reply:
Right :smile:
```python
    extras_require={'test': tests_requires, 'completion': ['shtab']},
```